### PR TITLE
Zoey_Completed html-web-page

### PR DESCRIPTION
**Pull request by Zoey1318**
html-web-page practice

### CODE REVIEW

File: html-web-page/index.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        <td>$50</td>
+        <td>$60</td>
+      </tr>
+    </tbody>
+  </table>
+
+  <p>Like what you see? Become a part of the JOTMC family today!</p>
+  <form action="index.html">
+    <label for="yourname"></label>
+    <input type="text" id="yourname" name="username" placeholder="Your Name">
```
Comment (scrumptiousdev) [inline html-web-page/index.html:87]: We are using `placeholder` in place of `label` so we don't necessarily have to have `label` and `id` for this one

File: html-web-page/index.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+    <label for="select-plan"></label>
+    <select name="selectplan" id="select-plan">
```
Comment (scrumptiousdev) [inline html-web-page/index.html:90]: Same goes here too. `label` is not needed hence no need for `id` as well

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]
Overall great job! We just need to clean up few things!

**Comment by Zoey1318** [comment]
recommit_html-web-page

**Review by scrumptiousdev — APPROVED** [review]
🪄 Approved!

**Pull request merged**